src/Faker/ORM/CakePHP/EntityPopulator.php

```diff
@@ -112,7 +113,7 @@ public function guessModifiers()
                     throw new \Exception(sprintf('%s belongsTo %s, which seems empty at this point.', $this->getTable($this->class)->table(), $assoc->table()));
                 }
 
-                $foreignKey = $foreignKeys[array_rand($foreignKeys)];
+                $foreignKey = Extension\Helper::randomElement($foreignKeys);
```

Comment:
Do not think this is wise here since the helper is experimental and this is not.

Or are we pushing this to 2.0? In that case it is fine, the orm plugin can just be removed

Reply:
@pimjansen 

It’s a step for `2.0`.